src/components/RestaurantList.tsx

```diff
@@ -0,0 +1,23 @@
+import RestaurantCard from './RestaurantCard';
+import { getRestaurants } from '@/api/restaurants';
+
+async function RestaurantList() {
```

Comment:
mä en tykkää function sanasta 😄 , pitäiskö käyttää vaa arrow funkkareita

```suggestion
const RestaurantList = async () => {
```

Reply:
Mulle käy kumpi vaan

ks. https://github.com/tperamaki/yala/pull/28

Reply:
mergesin ton ✅